R/FilterState.R

```diff
@@ -564,13 +564,15 @@ FilterState <- R6::R6Class( # nolint
     # @return a character string representation of a subset call
     #         that extracts the variable from the dataset
     get_varname_prefixed = function(dataname) {
+      varname <- private$get_varname()
+      varname_backticked <- sprintf("`%s`", varname)
```

Comment:
what do you think of making it conditional?
```suggestion
      if (make.names(varname) != varname) {
        varname <- sprintf("`%s`", varname)
      }
```

Reply:
The matrix case needs a backtick-free string, so you have to have both.